pkg/experiment/metastore/fsm/boltdb.go

```diff
@@ -53,9 +53,15 @@ func (db *boltdb) open(readOnly bool) (err error) {
 	}
 
 	opts := *bbolt.DefaultOptions
+	// open is called with readOnly=true to verify the snapshot integrity.
```

Comment:
I think we call open with readOnly=true and readOnly=false in different contexts, can we expand this comment to make that more clear?